experimental/addons/redis-exporter/README.md

```diff
@@ -0,0 +1,193 @@
+# Redis Exporter
+
+Prometheus Redis metrics exporter.
+> Supports Redis 2.x, 3.x, 4.x, 5.x, 6.x, and 7.x
+
+## Installation
+
+```shell
+vela addon enable redis-exporter
+```
+
+## Usage
+
+### Install relevant addons
+
+> If you have already installed the `prometheus-server` and `grafana` , please skip these steps to get started.
+
+1. Install the prometheus-server addon
+
+```shell
+vela addon enable prometheus-server
+```
+
+2. Install the grafana addon
+
+```shell
+vela addon enable grafana
+```
+
+3. Access your grafana through port-forward
+
+```shell
+kubectl port-forward svc/grafana -n o11y-system 8080:3000
+```
+
+Now you can access your grafana by access `http://localhost:8080` in your browser. The default username and password are `admin` and `kubevela` respectively.
+
+> You can change it by adding adminUser=super-user adminPassword=PASSWORD to step 3.
+
+
+### Work as a component
+
+#### 1. Create Redis Instance
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: redis
+  namespace: default
+spec:
+  components:
+    - name: redis
+      type: helm
+      properties:
+        repoType: "helm"
+        url: "https://charts.bitnami.com/bitnami"
+        chart: "redis"
+        version: "17.7.3"
+        values:
+          global:
+            redis:
+              password: <password>
+          master:
+            persistence:
+              size: 16Gi
+          replica:
+            persistence:
+              size: 16Gi
+
+```
+
+#### 2. Create Redis Secret
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <redis-secret-name>
+data:
+  username: <username-bash64> # default: ""
+  password: <password-bash64> # default: password in base64
+```
+
+#### 3. Create Redis Exporter Server
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: redis
+  namespace: default
+spec:
+  components:
+    - type: redis-exporter-server
+      name: redis-exporter
+      properties:
+        address: <host>:<port>
+        secretName: <redis-secret-name>
+        disableAnnotation: false
+        name: redis-server-exporter
+  traits:
+    - type: prometheus-scrape
+      properties:
+        port: <port>
+        path: /metrics
+  workflow:
+    steps:
+      - type: deploy
+        name: deploy-redis-exporter
```

Comment:
according to this issue https://github.com/kubevela/kubevela/issues/5670 this might not work.

please also update the app example below to add the deploy step.

Reply:
Ok, I have fixed it.